webpackjs/components/About/articles-data.js

```diff
@@ -0,0 +1,26 @@
+export const articles = [
```

Comment:
this is where the list of articles is currently coming from. We could edit this file to add new article objects. Or we could replace this with a proper CMS backend endpoint one day.

Reply:
i dig it